src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+/**
+ * A Task which has a description and a deadline.
+ *
+ * The deadline is of type LocalDateTime and is used to specify when the Task is due.
+ * Has a tag D.
+ */
+public class Deadline extends Task{
+
+    protected LocalDateTime deadline;
```

Comment:
Is there a reason for this to be protected rather than private? I noticed this pattern in a few other classes too.